fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanApplicationTerms.java

```diff
@@ -1773,7 +1773,7 @@ private void updateRecurringMoratoriumOnPrincipalPeriods(Integer periodNumber) {
 
     }
 
-    public void setTotalPrincipalAccounted(Money totalPrincipalAccounted) {
+    public void setTotalPrincipalAccountedForInterestCalculation(Money totalPrincipalAccounted) {
```

Comment:
Why is the name of the method a problem?

Reply:
There is also a `totalPrincipalAccounted` field in this object, so the original name of the setter is misleading, and does not do what says.

Reply:
... makes sense